Example of "otel-cli exec" crashes

`opel-cli exec --help` gives the following as example:

```
otel-cli exec -s "outer span" 'otel-cli exec -s "inner span" sleep 1'
```

Executing that triggers a crash:

```session
$ otel-cli exec -s "outer span" 'otel-cli exec -s "inner span" sleep 1'
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x9d756f]

goroutine 1 [running]:
github.com/equinix-labs/otel-cli/otelcli.doExec(0xc0001acf00?, {0xc0002657d0, 0x1, 0x3})
        github.com/equinix-labs/otel-cli/otelcli/exec.go:149 +0x102f
github.com/spf13/cobra.(*Command).execute(0xc000266c08, {0xc000265770, 0x3, 0x3})
        github.com/spf13/cobra@v1.8.0/command.go:987 +0xa91
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001a5b08)
        github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        github.com/spf13/cobra@v1.8.0/command.go:1032
github.com/equinix-labs/otel-cli/otelcli.Execute({0xb50aed, 0x7})
        github.com/equinix-labs/otel-cli/otelcli/root.go:89 +0x17d
main.main()
        github.com/equinix-labs/otel-cli/main.go:17 +0x45
```

--- a/otelcli/exec.go
+++ b/otelcli/exec.go
@@ -30,7 +30,7 @@ Examples:
 
 otel-cli exec -n my-cool-thing -s interesting-step curl https://cool-service/api/v1/endpoint
 
-otel-cli exec -s "outer span" 'otel-cli exec -s "inner span" sleep 1'`,
+otel-cli exec -s "outer span" -- otel-cli exec -s "inner span" sleep 1`,
 		Run:  doExec,
 		Args: cobra.MinimumNArgs(1),
 	}

